test/unit/effects.js

```diff
@@ -1028,7 +1028,7 @@ jQuery.each( {
 				jQuery( elem ).remove();
 
 			} );
-			this.clock.tick( 50 );
+			this.clock.tick( 100 );
```

Comment:
> Single-argument jQuery.when becomes necessarily asynchronous as a side effect

It feels to me like placing a distinction between single argument and multiple argument is a really bad idea. I opened #3100 about this.